proposals/repo-to-tar/README.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Building tarred bundles without being dependent on a OCI registry"
+authors: [ "Ashish Kumar <ashishndiitr@gmail.com>" ]
+status: "draft"
```

Comment:
```suggestion
status: "review"
```

Reply:
Shall I remove open as well as answered questions section or maybe we can keep them for future reference since they are answered in the answered section ?  What is the correct standard to follow ?